src/natcap/invest/urban_nature_access.py

```diff
@@ -337,11 +350,179 @@ def execute(args):
             greenspace_population_ratio_task,
         ])
 
+    # SUP_DEMi_cap
```

Comment:
Does this shorthand comment for the following operation stem from the design doc or refer to something else in code? Guess I'm just curious if this comment is helpful!

Reply:
Yeah, it's from the design doc.  I made a note about it in a2b4e2c5